Makefile

```diff
@@ -70,13 +70,10 @@ manifests: controller-gen kustomize yq ## Generate WebhookConfiguration, Cluster
 .PHONY: generate
 generate: controller-gen conversion-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
```

Comment:
Please remove dependencies to conversion gen.

Reply:
[19345ab](https://github.com/cybozu-go/moco/pull/602/commits/19345abaf9f37b3673a636485ac3dcf38db92e0f)